app/scripts/lib/validate.js

```diff
@@ -103,7 +103,41 @@ define([
       }
 
       return password.length >= Constants.PASSWORD_MIN_LENGTH;
+    },
+
+    /**
+     * Check if a value is not falsy.
+     *
+     * @return the value if it's not falsy, the defaultValue otw.
+     */
+    isNotFalsy: function (value, defaultValue) {
+      return value || defaultValue;
+    },
+
+    /**
+     * Check if at least one of the arguments is not falsy.
+     *
+     * @return the first non falsy argument if there is one, the last argument
+     * otw.
+     */
+    areNotFalsy: function () {
+      for (var i = 0; i < arguments.length - 1; i++){
```

Comment:
Also, can you add a quick comment why you use `i < arguments.length - 1` instead of `i < arguments.lenght`, otherwise a casual reader might think this is an error.